src/GitHub.App/ViewModels/PullRequestListViewModel.cs

```diff
@@ -23,6 +24,40 @@ namespace GitHub.ViewModels
     [PartCreationPolicy(CreationPolicy.NonShared)]
     public class PullRequestListViewModel : BaseViewModel, IPullRequestListViewModel, IDisposable
     {
+        // this is the order that will show up in the dropdown
+        static IReadOnlyList<PullRequestSortOrder> sortOrders = new List<PullRequestSortOrder> {
+            new PullRequestSortOrder {
+                SortOrder = SortOrder.UpdatedDescending,
+                Name = Resources.SortOrderUpdatedDescending,
+                Comparer = OrderedComparer<IPullRequestModel>.OrderByDescending(x => x.UpdatedAt)
+            },
+            new PullRequestSortOrder {
+                SortOrder = SortOrder.CreatedDescending,
+                Name = Resources.SortOrderCreatedDescending,
+                Comparer = OrderedComparer<IPullRequestModel>.OrderByDescending(x => x.CreatedAt)
+            },
+            new PullRequestSortOrder {
+                SortOrder = SortOrder.UpdatedAscending,
+                Name = Resources.SortOrderUpdatedAscending,
+                Comparer = OrderedComparer<IPullRequestModel>.OrderBy(x => x.UpdatedAt)
+            },
+            new PullRequestSortOrder {
+                SortOrder = SortOrder.CreatedAscending,
+                Name = Resources.SortOrderCreatedAscending,
+                Comparer = OrderedComparer<IPullRequestModel>.OrderBy(x => x.CreatedAt)
+            }
+        };
+
+        // a helper lookup table for easy access to the sorting comparers
+        static Dictionary<SortOrder, PullRequestSortOrder> sortOrderIndex;
+
+        // TODO: Move strings to resources
+        IReadOnlyList<PullRequestState> states = new List<PullRequestState> {
```

Comment:
So this is just casting a mutable list to an `IReadOnlyList` which can be cast back to `List`. I know we probably won't do that, but I'm thinking of the next person to come along. The safe thing to do is to wrap `List` with a `ReadOnlyList`. We have an extension method `ToReadOnlyList` for observables. Wouldn't be hard to write one for `IEnumerable`.